Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1745,6 +1745,12 @@ def lhegensim2018ml(fragment,howMuch):
 
 steps['DIGIPRMXUP15_PU50']=merge([digiPremixUp2015Defaults50ns])
 steps['DIGIPRMXUP17_PU25']=merge([digiPremixUp2017Defaults25ns])
+steps['DIGIPRMXUP17_PU25_RD']=merge([digiPremixUp2017Defaults25ns,
+                                     {
+                                       '--customise_commands':"\"process.EcalLaserCorrectionServiceMC = cms.ESProducer('EcalLaserCorrectionServiceMC') \\n process.GlobalTag.toGet = cms.VPSet( cms.PSet(     record = cms.string('EcalLaserAPDPNRatiosMCRcd'),  tag = cms.string('EcalLaserAPDPNRatios_UL_2017_mc'),  connect = cms.string('frontier://FrontierProd/CMS_CONDITIONS') ) )\""
```

Comment:
@amassiro , didn't mean to be picky here but could you reduce some lines to have similar format as others? 
steps['DIGIPRMXUP17_PU25_RD']=merge({'--customise_commands':"\"process.EcalLaserCorrectionServiceMC = cms.ESProducer('EcalLaserCorrectionServiceMC') \\n process.GlobalTag.toGet = cms.VPSet(cms.PSet(record = cms.string('EcalLaserAPDPNRatiosMCRcd'), tag = cms.string('EcalLaserAPDPNRatios_UL_2017_mc'), connect = cms.string('frontier://FrontierProd/CMS_CONDITIONS')))\""},[digiPremixUp2017Defaults25ns])

Reply:
Sure, I can put everything in one single line.
Committing it right now ...